Sources/Recycling/ListViews/UserAds/Cell/UserAdsListViewCell.swift

```diff
@@ -250,3 +248,11 @@ public class UserAdsListViewCell: UITableViewCell {
         }
     }
 }
+
+extension UserAdsListViewCell: ImageLoading {
```

Comment:
Change to `public extension UserAdsListViewCell` than you don't need the `public` modifier in front of each function.